packages/chain-adapters/src/cosmossdk/cosmos/CosmosChainAdapter.ts

```diff
@@ -63,7 +63,12 @@ export class ChainAdapter extends CosmosSdkBaseAdapter<KnownChainIds.CosmosMainn
   }
 
   async getAddress(input: GetAddressInput): Promise<string> {
-    const { wallet, bip44Params = ChainAdapter.defaultBIP44Params, showOnDevice = false } = input
+    const { wallet, bip44Params, showOnDevice = false } = input
+
+    if (!bip44Params) {
+      throw new Error('bip44Params are required in Cosmos SDK getAddress')
+    }
```

Comment:
This will go away in a follow-up PR, making `bip44Params` required will bork swappers